### ISSUE TITLE
v2.0.26 release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/arizona-bootstrap#1202]
Non devDependency releated PRs merged since v2.0.25:
- #1005
- #1075 
- #1107 
- #1124 
- #1150 

Full list of 2.x branch commits since 2.0.25: https://github.com/az-digital/arizona-bootstrap/compare/v2.0.25...2.x

PRs that would be good to merge before a new release:
- #1198

**Comment by joeparsons** [comment]
## Release notes snippet:

`git log v2.0.25..HEAD --pretty='format:- %s' `
```
- Closes #1192: Add the CSS suggestions to avoid accordion titles overlapping arrows. (#1198)
- Bump the development-dependencies group with 7 updates (#1191)
- Bump the development-dependencies group with 4 updates (#1187)
- Bump the development-dependencies group with 3 updates (#1184)
- Bump @babel/preset-env from 7.23.9 to 7.24.0 (#1176)
- Bump eslint from 8.56.0 to 8.57.0 (#1168)
- Bump @babel/core from 7.23.9 to 7.24.0 (#1175)
- Bump autoprefixer from 10.4.17 to 10.4.18 (#1173)
- Bump nodemon from 3.0.3 to 3.1.0 (#1166)
- Bump hugo-bin from 0.119.0 to 0.120.7 (#1174)
- Closes #1148 Add twtter x icon to bootstrap docs. (#1150)
- Bump stylelint-config-twbs-bootstrap from 13.0.0 to 14.0.0 (#1137)
- Bump rollup from 4.9.6 to 4.12.0 (#1151)
- Bump terser from 5.27.0 to 5.27.1 (#1156)
- Bump eslint-plugin-unicorn from 50.0.1 to 51.0.1 (#1141)
- Bump postcss from 8.4.33 to 8.4.35 (#1142)
- Bump eslint-config-xo from 0.43.1 to 0.44.0 (#1139)
- Bump stylelint from 16.2.0 to 16.2.1 (#1136)
- Bump hugo-bin from 0.118.1 to 0.119.0 (#1129)
- Bump linkinator from 6.0.3 to 6.0.4 (#1109)
- Closes #1122: Add callout variants for all theme colors. (2.x) (#1124)
- Closes Issue #1105: backport dev dependency updates to the 2.x branch. (#1107)
- Closes #1073: Update old brand links to the new marcom site. (#1075)
- Bump @babel/preset-env from 7.23.2 to 7.23.5 (#1051)
- Bump @babel/core from 7.23.2 to 7.23.5 (#1054)
- Bump rollup from 4.1.5 to 4.6.1 (#1049)
- Bump terser from 5.22.0 to 5.24.0 (#1018)
- Bump hugo-bin from 0.115.0 to 0.116.4 (#1027)
- Bump eslint from 8.52.0 to 8.54.0 (#1037)
- Bump linkinator from 6.0.0 to 6.0.2 (#1046)
- Bump @babel/cli from 7.23.0 to 7.23.4 (#1047)
- Bump stylelint-config-twbs-bootstrap from 11.0.1 to 12.0.0 (#1048)
- Bump eslint-plugin-unicorn from 48.0.1 to 49.0.0 (#1009)
- Bump rollup from 4.1.4 to 4.1.5 (#1008)
- Bump linkinator from 5.0.2 to 6.0.0 (#1007)
- Closes #1004: backport callout border color changes. (#1005)
- Bump stylelint from 15.10.3 to 15.11.0 (#994)
- Bump eslint from 8.51.0 to 8.52.0 (#995)
- Bump terser from 5.21.0 to 5.22.0 (#996)
- Bump rollup from 4.0.2 to 4.1.4 (#992)
- Closes #987: bump dev dependency versions on the 2.x branch. (#989)
- Bump glob from 10.3.4 to 10.3.10 (#966)
- Bump terser from 5.19.4 to 5.20.0 (#948)
- Bump eslint from 8.49.0 to 8.50.0 (#947)
- Bump @babel/preset-env from 7.22.15 to 7.22.20 (#937)
- Bump jquery from 3.7.0 to 3.7.1 (#918)
- Bump postcss from 8.4.29 to 8.4.31 (#963)
- Bump @babel/core from 7.22.15 to 7.22.17 (#923)
- Bump hugo-bin from 0.113.0 to 0.114.2 (#917)
- Bump eslint from 8.48.0 to 8.49.0 (#925)
- Bump linkinator from 5.0.1 to 5.0.2 (#904)
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/arizona-bootstrap/releases/tag/v2.0.26
